src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/XACommitNodesHandler.java

```diff
@@ -347,4 +381,18 @@ public void debugCommitDelay() {
         }
 
     }
+
+    public void waitUntilSendFinish() {
+        try {
+            this.lockForErrorHandle.lock();
+            if (!this.sendFinishedFlag) {
```

Comment:
change if to while

Reply:
it's OK to use if here,the condition will  only be signaled for once